### PR TITLE
[16.0][MIG] base_report_to_printer_mail: Migration to 16.0

### PR DESCRIPTION
**Pull request by QuocDuong1306**
Depend on #332

### CODE REVIEW

**Review by BT-anieto — COMMENTED** [review]
Thanks for the PR. You would need to squash the commits as indicated in https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by QuocDuong1306** [comment]
Hi @BT-anieto , I updated

**Review by BT-anieto — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

File: setup/base_report_to_printer_mail/odoo/addons/base_report_to_printer_mail [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../../base_report_to_printer_mail
```
Comment (sebalix) [inline setup/base_report_to_printer_mail/odoo/addons/base_report_to_printer_mail:1]: change related to `setup` folder are also normally part of the `[IMP] $module: pre-commit stuff` if you follow the OCA migration guidelines: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0#technical-method-to-migrate-a-module-from-150-to-160-branch

Comment (QuocDuong1306) [reply setup/base_report_to_printer_mail/odoo/addons/base_report_to_printer_mail:1]: Thank for your review, I updated

**Comment by sebalix** [comment]
@QuocDuong1306 thank you!

/ocabot merge nobump

**Pull request merged**